silkrpc/core/gas_price_oracle_test.cpp

```diff
@@ -343,7 +343,7 @@ TEST_CASE("suggested price") {
     }
 
     SECTION("when there are 60 blocks with with 0x07 base fee with fee == kDefaultMaxPrice") {
-        FixedBlockData data = {0x07, kDefaultMaxPrice, kDefaultMaxPrice, kDefaultMaxPrice, kDefaultMaxPrice};
+        FixedBlockData data = {0x07, kDefaultMaxPrice+0x07, kDefaultMaxPrice+0x07, kDefaultMaxPrice+0x07, kDefaultMaxPrice+0x07};
```

Comment:
`kDefaultMaxPrice+0x07` -> `kDefaultMaxPrice + 0x07` to keep code more uniform

Reply:
ok